### DISCUSSION TITLE
Bundled Releases of cFS and Supported Apps?

### DISCUSSION THREAD

**Discussion by jeffstjean** [nasa/cFS#522] [Q&A] [answered]
Does GSFC provide bundled releases of the cFE core (OSAL and PSP) accompanied with compatible applications or some other way of ensuring the correct version of each app is cloned? I know some apps haven't yet been publicly updated to some of the most recent cFE versions so I just want to make sure I download the correct versions.

**Comment by skliper** [comment, marked as answer]
I've tried to indicate in the release notes the app compatibility.  We've adopted a development cycle naming scheme, for example everything that says Draco-rc1 should be compatible.  CF, CS, DS, FM, LC, HK, HS, MD, MM, SC are all up-to-date and include CI that at least compiles, runs the unit test and does a simple startup confirmation w/ the main framework.  We've got plans to add a CI run that adds all the up-to-date apps from the bundle repo, but we haven't had time to implement that yet.

**Reply by jeffstjean** [reply]
Awesome thanks! For new projects, is Draco-rc1 the recommended place to start?

**Reply by skliper** [reply]
Draco-rc1 is a good starting point if you need to use a tag, or Draco-rc2 just showed up in most repos although I haven't worked on the paperwork yet.  That said, I would track main unless you really need to lock it down now or if something is introduced that you don't want.  Our CI has improved over the years to where main is really pretty solid.  Draco-rc2 is just a slight improvement (better code coverage in the  unit tests mostly, but minor other cleanup and a few requirements updates), and I currently anticipate Draco-rc3 the end of this summer to be where we initiate the next official release.

**Reply by jeffstjean** [reply]
Everything worked like a charm @skliper thanks! Has there been any talk around upgrading SBN or what's the state of that application? We're planning on having two cFS instances running on separates processes and don't want to roll our own solution if there's something already out there.

**Reply by skliper** [reply]
Good questions about SBN.  Looks like it was updated recently for Draco typesafety compliance, but I'm not aware of any plans for updated releases.  Looks like main branch might work though. @CDKnightNASA is probably the best contact for further information.

**Comment by mprather** [comment]
Maurice Prather
President, Okean Solutions, Inc.
E: ***@***.******@***.***>, P: 206.383.0181
***@***.***
MONSID®<http://monsid.com/> fault management software
